api/rpc/stat/stat.go

```diff
@@ -32,6 +32,13 @@ func New(cfg Config) (*Stat, error) {
 // ExecuteQuery implements business logic for StateServer interface
```

Comment:
Definitely a typo, should say StatServer, but I think it is current per the proto.pb.go files declaration:
```

type StatServer interface {
	ExecuteQuery(context.Context, *QueryRequest) (*QueryReply, error)
}
```
Am I missing something?